cblox/include/cblox/core/submap_collection.h

```diff
@@ -46,9 +46,10 @@ class SubmapCollection {
   // exist... This puts the onus on the caller to call exists() first. I don't
   // like this but I can't see a solution.
   const SubmapType &getSubMap(const SubmapID submap_id) const;
-  // Note(alexmillane): Unlike the above this function returns a nullptr when
-  // the map doesn't exist. No hard crash.
-  typename SubmapType::ConstPtr getSubMapConstPtrById(
+  // Note(alexmillane): Unlike the above method, the two methods below return a
+  // nullptr when the map doesn't exist. No hard crash.
+  typename SubmapType::Ptr getSubMapPtr(const SubmapID submap_id);
```

Comment:
I totally agree with the naming change, that had actually annoyed me as well. Let's change it now while this only affects a few people.

Reply:
@victorreijgwart One additional naming inconsistency that is annoying me is the varying treatment of the word "submap". We (I) are treating submap as one word most places (e.g. submap_id) but two words in camelcase (e.g. getSubMapPtr). I think we should also make this consistent in the near future.

Reply:
@alexmillane Yes I agree, it'd be great if Submap / SubMap was capitalized in a consistent way.
Changing it should be quick. I propose to do it now and push it to this PR.
Which variant do you prefer? I'd personally vote for Submap since it's a bit easier on the eyes (less chunks) and faster to write. But it's definitely your decision :)

Reply:
Ok, if you wanna do it here, that would be great!

yeah, I totally agree with you; let's treat submap as one word from now on such that SubMap -> Submap

Reply:
Done!